### ISSUE TITLE
Goto is not supported

### ISSUE THREAD

**Issue by KayMKM** [microsoft/BotFramework-Composer#2375]
## Is your feature request related to a problem? Please describe.
Goto is not supported in current composer

**Comment by KayMKM** [comment]
there is an scenario to use goto: in calendar skill, the confirm creating step is a loop. the bot will confirm whether user want to change any slot until user confirm to create meeting.
Currently I put the confirm logic in a dialog, so I need to pass all related parameters into confirm dialog. If we can use goto it will be easy to repeat confirm flow just in create meeting dialog

**Comment by cwhitten** [comment]
@KayMKM For my own knowledge, what is keeping the `Repeat this dialog` action from working in your scenario?

**Comment by KayMKM** [comment]
@cwhitten When I want to only repeat part of logic in a dialog. The repeat this dialog will repeat the whole dialog.

**Comment by hibrenda** [comment]
@mareekuh @sangwoohaan need your suggestion.

**Comment by mareekuh** [comment]
@sangwoohaan - could you take a pass at what a 'goto' node would look like?
and @DesignPolice - when Sang is done, please take a visual pass to make it fit in with the Fluent language.

**Comment by mareekuh** [comment]
@sangwoohaan  and @DesignPolice  - should we make GOto a special node or have an indicator (check PVA)?

**Comment by mewa1024** [comment]
Is this the same as Microsoft.GotoAction in the sdk.schema?

```
    "Microsoft.GotoAction": {
            "$role": "implements(Microsoft.IDialog)",
            "title": "Go to Action",
            "description": "Go to an an action by id.",
            "type": "object",
            "properties": {
                "$kind": {
                    "title": "$kind",
                    "description": "Defines the valid properties for the component you are configuring (from a dialog .schema file)",
                    "type": "string",
                    "pattern": "^[a-zA-Z][a-zA-Z0-9.]*$",
                    "const": "Microsoft.GotoAction"
                },
                "$copy": {
                    "title": "$copy",
                    "description": "Copy the definition by id from a .dialog file.",
                    "type": "string",
                    "pattern": "^(([a-zA-Z][a-zA-Z0-9.]*)?(#[a-zA-Z][a-zA-Z0-9.]*)?)$"
                },
                "$id": {
                    "title": "$id",
                    "description": "Inline id for reuse of an inline definition",
                    "type": "string",
                    "pattern": "^([a-zA-Z][a-zA-Z0-9.]*)$"
                },
                "$designer": {
                    "title": "$designer",
                    "type": "object",
                    "description": "Extra information for the Bot Framework Designer."
                },
                "id": {
                    "type": "string",
                    "title": "Id",
                    "description": "Optional id for the dialog"
                },
                "disabled": {
                    "$role": "expression",
                    "title": "Disabled",
                    "description": "Optional condition which if true will disable this action.",
                    "examples": [
                        "user.age > 3"
                    ],
                    "oneOf": [
                        {
                            "type": "boolean",
                            "title": "boolean"
                        },
                        {
                            "type": "string",
                            "title": "Expression",
                            "description": "Expression evaluating to boolean."
                        }
                    ]
                },
                "actionId": {
                    "$role": "expression",
                    "type": "string",
                    "title": "Action Id",
                    "description": "Action Id to execute next"
                }
            },
            "additionalProperties": false,
            "patternProperties": {
                "^\\$": {
                    "type": "string"
                }
            },
            "anyOf": [
                {
                    "title": "Reference",
                    "required": [
                        "$copy"
                    ]
                },
                {
                    "title": "Type",
                    "required": [
                        "actionId",
                        "$kind"
                    ]
                }
            ]
        },
```

**Comment by luhan2017** [comment]
@mewa1024, yes exactly.

**Comment by arafattehsin** [comment]
Hi team! I noticed that instead of opening up an issue, we've got a similar requirement as above. At times, in order to test certain conversation block, I don't want to go through the whole conversation, is it possible to have GoTo action where we just provide an ID and then it only tests that part? 

As @mareekuh mentioned, PVA already has similar approach but I understand as it is only related to the topic. 

Can we have something similar (or better) here? Also, is there a way to achieve it right now with any workaround?
